test/uk/gov/hmrc/helptosavefrontend/controllers/email/AccountHolderUpdateEmailAddressControllerSpec.scala

```diff
@@ -49,18 +49,18 @@ class AccountHolderUpdateEmailAddressControllerSpec extends AuthSupport {
   val mockNSIConnector = mock[NSIConnector]
 
   def mockEnrolmentCheck(input: NINO)(result: Either[String, EnrolmentStatus]): Unit =
```

Comment:
doesn't look like we need the NINO here now

Reply:
fixed